opte/src/engine/arp.rs

```diff
@@ -11,23 +11,65 @@
 /// * RFD 9 -- NETWORKING CONSIDERATIONS
 /// ** §1.13 ARP
 /// * RFC 826 -- An Ethernet Address Resolution Protocol
-use super::ether::{EtherHdr, EtherMeta, ETHER_TYPE_ARP, ETHER_TYPE_IPV4};
-use super::headers::{Header, RawHeader};
-use super::packet::{
-    Packet, PacketMeta, PacketRead, PacketReader, PacketWriter, Parsed,
-    ReadErr, WriteError,
-};
-use super::rule::{
-    AllowOrDeny, ArpHtypeMatch, ArpOpMatch, ArpPtypeMatch, DataPredicate,
-    EtherAddrMatch, EtherTypeMatch, GenPacketResult, HairpinAction,
-    Ipv4AddrMatch, Payload, Predicate,
-};
+/// Address Resolution Protocol
+///
+/// Relevant Docs
+///
+/// * RFD 9 -- NETWORKING CONSIDERATIONS
+/// ** §1.13 ARP
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
+/// Address Resolution Protocol
+///
+/// Relevant Docs
+///
+/// * RFD 9 -- NETWORKING CONSIDERATIONS
+/// ** §1.13 ARP
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
+/// Address Resolution Protocol
+///
+/// Relevant Docs
+///
+/// * RFD 9 -- NETWORKING CONSIDERATIONS
+/// ** §1.13 ARP
+/// * RFC 826 -- An Ethernet Address Resolution Protocol
```

Comment:
This is still all messed up. At the top of the file, it should have just been:

```
/ Address Resolution Protocol
 ///
 /// Relevant Docs
 ///
 /// * RFD 9 -- NETWORKING CONSIDERATIONS
 /// ** §1.13 ARP
 /// * RFC 826 -- An Ethernet Address Resolution Protocol
```

(Yes, the coment is kind of crap, but I'm just trying to point out fmt has gone off the rails here).

Reply:
Weird, thanks. I'll look again.

Reply:
Oh I think I see what happened. That comment was attached to a nested import, and `cargo fmt` must have duplicated it when it expanded each item to its own line. I'll remove the extra copies.